index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/normalize.css">
+    <link rel="stylesheet" href="/css/styles.css">
+    <title>PC Store</title>
+</head>
+
+<body>
+    <div class="bar">
+        <header>
+            <div class="logo">
+                <img class="logo-image" src="/img/logo.png" alt="">
+                <p class="logo-name">GAMING WORLD</p>
+            </div>
+            <nav>
+                <a href="products.html">Our Products</a>
+            </nav>
+
+        </header>
+    </div>
+    <section class="pc-sell">
+        <img class="pc-image" src="/img/Gaming-Computer-PNG-Photo.png" alt="">
+        <p class="pc-info">
```

Comment:
for this section i think it would be better if you use a <ul>/<li> tag for the description of the product

Reply:
great job! you used a polite to gently point out places of improvement. it will be nice if you could point out reasons why the use of ul and li tag is better. This will help the developer to understand why you are requesting changes.

Reply:
thank you, I think the reason is that when it comes to a description for best practices is better to use "ul" "li" tags, I also forgot to mention that for the price it would look better if it would has some styles like a bolder font, and for the logo it would be better to add an "a href" tag as a home button.